prisma/migrations/20240821072645_rename_lnbits_pk/migration.sql

```diff
@@ -0,0 +1,15 @@
+/*
+  Warnings:
+
+  - The primary key for the `WalletLNbits` table will be changed. If it partially fails, the table could be left without primary key constraint.
+  - You are about to drop the column `int` on the `WalletLNbits` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "WalletLNbits" RENAME COLUMN "int" TO "id";
+
+UPDATE "Wallet"
+SET wallet = to_jsonb(NEW)
+WHERE id IN (
+  SELECT "walletId" FROM "WalletLNbits"
+)
```

Comment:
Is this valid sql? What does NEW refer to in an update statement?

Reply:
It wasn't. I fixed it.

Reply:
Whoops, I remember I ran an update query manually but must have forgotten to check what I committed